lib/parse.js

```diff
@@ -7,6 +7,6 @@ function parse (profile, graph, uri, mimeType, callback) {
     $rdf.parse(profile, graph, uri, mimeType)
     return callback(null, graph)
   } catch(e) {
-    return callback('Cound not load/parse profile data')
+    return callback('Cound not load/parse profile data', e)
```

Comment:
Oh yes good point, it is better for callbacks to return errors rather than string.  But that is how the code was before.

I had assumed on reading the error string, it was a just a `console.log` or `debug`, my bad.

Tests are broken already for this module I believe it references an account on databox that no longer works.

I've now traced through the debug on this, so probably cold just close this PR, or changing the strong to new Error()